test/performance/odp_sched_perf.c

```diff
@@ -99,6 +104,8 @@ typedef struct test_global_t {
 	test_stat_t stat[ODP_THREAD_COUNT_MAX];
 	thread_arg_t thread_arg[ODP_THREAD_COUNT_MAX];
 	odp_atomic_u32_t num_worker;
+	/* Break worker loop if set to 1 */
```

Comment:
Actually, this breaks the loop if nonzero. Maybe the comment could just be dropped as the code is obvious anyway.

Reply:
Comment dropped in V2.